### DISCUSSION TITLE
how can I ask attendees if they want to subscribe to my mailing list ?

### DISCUSSION THREAD

**Discussion by pfabre31** [alfio-event/alf.io#1393] [General]
Hi everyone, first thank you so much for developing this. I will use it for the first tour of my band an it's of great help.

I would simply want to have ask my attendees if they want to subscribe to my mailing list during the reservation process. I know I can do that by adding a simple field -I used to be a web developer so I can do that-, but as I mentionned in another discussion channel, for some reason I can't compile Java locally and therefore send a customized version to my server -heroku.

What would be the easiest way to proceed knowing that ? Polls ? I created a poll but I cannot seem to know how that works -where can attendees access them.

Thanks a lot !

**Comment by syjer** [comment]
Hi @pfabre31 , in alf.io you can add custom fields in your reservation.

In the admin, you can go to the section "Attendees data to collect", click on "add new field" and fill like this (you need to add a "checkbox" type field with a single value "true" defined:

![Screenshot from 2024-08-27 16-10-29](https://github.com/user-attachments/assets/45819ecc-c0e6-46f3-a7ae-0c7657b28225)

You will notice that during the reservation, now under each ticket the following new field will be present:

![Screenshot from 2024-08-27 16-10-45](https://github.com/user-attachments/assets/a026a553-923c-4342-ba82-dcc48ea5d481)

The "Attendees data to collect" is quite flexible, you can notice we already have pre generated some often used use cases.

The data can be downloaded by clicking in the event admin : "Download" -> "Attendees data". You will need to select all the fields.
